force-app/main/default/permissionsets/HOT_Tolk_Formidler.permissionset-meta.xml

```diff
@@ -383,6 +388,11 @@
         <field>HOT_Request__c.IsAcute__c</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
+        <editable>false</editable>
```

Comment:
Disse skal vel ikke være med, har du tatt det ned fra en sandbox? I så fall bør du pushe, og retrieve fra en scratch org for å få det riktig.

Reply:
Tok det fra en scratch org. Er jo false false, så trenger jo ikke å være med, men om sfdx har oppdatert slik at den henter det ned hver gang..

Reply:
Snodig, de som er false/false pleier å bli fjernet fra xml-fila. Er jo ikke noe farlig.